liquibase-couchbase/src/test/java/liquibase/ext/couchbase/provider/ExpressionDocumentKeyProviderTest.java

```diff
@@ -0,0 +1,85 @@
+package liquibase.ext.couchbase.provider;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import com.couchbase.client.java.json.JsonObject;
+import liquibase.ext.couchbase.exception.ProvideKeyFailedException;
+import liquibase.ext.couchbase.types.TokenType;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class ExpressionDocumentKeyProviderTest {
+
+    @Test
+    void Should_parse_expression_correctly() {
+        String expression = "#token#%value_value%%%%#%#%#";
+        Map<String, TokenType> expected = new LinkedHashMap<>();
```

Comment:
Expected creating could be extracted to the separate method, or static constant

Reply:
Sorry, didn't get the meaning of separating creational of specific data for specific test case. That Map is used only in this one test among other 6 different unit tests. Are you sure that every specific data preparation for each test case (not suite) should be done in separate methods? I assume that if sonar does not mark such methods as too long, it means that there is no need to separate such logic.